subsystems/intakesubsystem.py

```diff
@@ -0,0 +1,54 @@
+from commands2 import SubsystemBase
+from wpilib import PWMVictorSPX, PneumaticHub, PneumaticsModuleType, RobotBase, Solenoid
+import constants
+
+
+class IntakeSubsystem(SubsystemBase):
+    def __init__(self) -> None:
+        SubsystemBase.__init__(self)
+        self.setName(__class__.__name__)
+        self.intakeDeployed = False  # default to intake retracted and not reverseed
+        self.intakeReversed = False
+
+        self.pneumaticsHub = PneumaticHub(1)
+
+        self.leftSolenoid = Solenoid(PneumaticsModuleType.REVPH, 0)
+        self.leftSolenoid.set(False)
+
+        self.rightSolenoid = Solenoid(PneumaticsModuleType.REVPH, 1)
+        self.rightSolenoid.set(False)
+
+        if RobotBase.isReal():
+            pass
+        else:
+            self.intakeMotor = PWMVictorSPX(constants.kSimIntakeMotorPort)
+
+    def toggleIntake(self) -> None:
+        self.intakeDeployed = not self.intakeDeployed
```

Comment:
these should actually move the solenoids, I recommend a general "deployed" getter function that uses if both of them are extended